cli/js/40_lint.js

```diff
@@ -345,6 +418,35 @@ export class Context {
   }
 }
 
+/**
+ * TODO(@marvinhagemeister): Is it worth it to do this lazily?
```

Comment:
Seems fine, I think most ppl won't be linting comments so better to do it lazily